src/single-spa-vue.js

```diff
@@ -51,28 +58,38 @@ function mount(opts, mountedInstances, props) {
       appOptions.el = props.domElement;
     }
 
-    if (!appOptions.el) {
+    let domEl;
+    if (appOptions.el) {
+      domEl = document.querySelector(appOptions.el);
```

Comment:
I think that the case when `props.domElement` is provided as raw dom element and `opts.appOptions.el` is `undefined` should be supported. Because otherwise f.e. `single-spa-react/parcel`, which provides direct reference to the element as `props.domElement`, will become incompatible.

Reply:
Ah good point I forgot that it was possible to pass in the dom element. I will update this. Thanks for the review